tests/unit/components/sl-chart-test.js

```diff
@@ -0,0 +1,121 @@
+import Ember from 'ember';
+import { moduleForComponent, test } from 'ember-qunit';
+
+var object = {
+    chartOptions: {
+        chart: {
+            type: 'bar'
+        },
+
+        xAxis: {
+            categories: [ 'Apples', 'Bananas', 'Oranges' ]
+        },
+
+        yAxis: {
+            title: {
+                text: 'Fruit Eaten'
+            }
+        }
+    }
+};
+
+var series = Ember.A([
+    { name: 'Alice', data: [ 1, 0, 4 ] }
+]);
+
+moduleForComponent( 'sl-chart', 'Unit | Component | sl chart', {
+	unit: true
+});
+
+test( 'Default classNames are present', function( assert ) {
+    this.subject({ object, series });
+
+    assert.ok(
+        this.$().hasClass('panel'),
```

Comment:
Add space after open parenthesis and before closing one.  See https://github.com/softlayer/ember-style-guide/blob/master/javascript.md#block-statements